### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by jinzequn** [zjuela/LapSRN-tensorflow#1]
I'm sorry that I don't understand the loss function in your code, it's different with paper.

**Comment by zjuela** [comment]
@jinzequn Thanks for pointing it out. I have changed the loss function from l2 to Charbonnier penalty. And there has some difference in the results after changing it.

**Comment by jinzequn** [comment]
The results become better or worse? I have never konw about Charbonnier penalty before, I wonder that does Charbonnier penalty really work?

**Comment by zjuela** [comment]
I did not do large quantitative evaluation, but visually they look slightly sharper comparing to l2 loss.

**Comment by jinzequn** [comment]
Thanks a lot!
